private/buf/cmd/buf/command/curl/curl.go

```diff
@@ -362,7 +373,40 @@ one is provided`,
 		userAgentFlagName,
 		"A",
 		"",
-		`The user agent string to send`,
+		`The user agent string to send. This is ignored if a -H or --header flag is provided
+that sets a header named 'User-Agent'.`,
+	)
+	flagSet.StringVarP(
+		&f.UserCreds,
+		userCredsFlagName,
+		"u",
+		"",
+		`The user credentials to send, via a basic authorization header. The value should be
+in the format "username:password". If the value has no colon, it is assumed to just be
+the username, in which case you will be prompted to enter a password. This overrides
+the use of a .netrc file. This is ignored if a -H or --header flag is provided that sets
+a header named 'Authorization'.`,
+	)
+	flagSet.BoolVarP(
+		&f.UseNetrc,
+		netrcFlagName,
+		"n",
+		false,
+		`If true, a file named .netrc in the user's home directory will be examined to find
+credentials for the request. The credentials will be sent via a basic authorization header.
+The command will fail if the file does not have an entry for the hostname in the URL. This
+flag is ignored if a -u or --user flag is present. This is ignored if a -H or --header flag
+is provided that sets a header named 'Authorization'.`,
+	)
+	flagSet.StringVar(
+		&f.NetrcFile,
+		netrcFileFlagName,
+		"",
+		`This is just like use -n or --netrc, except that the named file is used instead of
```

Comment:
Same comments about fmt.Sprintf, although tbh really consider this a nit, we don't need to do it if it takes too much time

Reply:
Done. I applied this everywhere... except the long paragraphs of content for the main help content. That references numerous flags, including many references in the examples. That... didn't seem worth it, especially if it meant moving the actual flag name references to far away from the format indicator. If Go had python- or kotlin-style interpolation, that would be another story, but...